libs/platform/src/lib/components/split-menu-button/split-menu-button.component.html

```diff
@@ -1,13 +1,15 @@
-<div class="fd-button-split" role="group" aria-label="button-split">
+<div class="fd-button-split" role="group" aria-label="button-split" [attr.dir]="dir">
     <button
         fd-button
         [attr.id]="id"
         [attr.name]="name"
-        aria-label="buttonLabel"
+        [attr.title]="title"
         [disabled]="disabled"
         [fdType]="type"
         [glyph]="icon"
+        tabindex="0"
         [compact]="contentDensity === 'compact'"
+        aria-label="buttonLabel"
         (click)="primaryButtonClicked($event)"
     >
```

Comment:
Can't we reuse <fdp-button> component?

Reply:
<img width="656" alt="Screenshot 2020-06-03 at 8 24 44 PM" src="https://user-images.githubusercontent.com/57661487/83652514-583af800-a5d8-11ea-9aa4-5c837ea2ec89.png">
using fdp-button does not work with styles defined for split-button in fundamental-styles.

Reply:
it shoudl be ok with native. if we don't need some extra features then let;s stick with native inside our components, It is also about the performance